CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Issues
+- Please use the templates. If none of them is a perfect match, choose the closest one.
+
+# Security vulnerabilities
+- If you discover a security vulnerability, please do not file an issue. Instead, contact the Intranet developers directly. If you have no other means of doing this, send an email to intranet [at] tjhsst [dot] edu.
+  - Even if you know how to fix the issue, please contact the Intranet developers directly before opening a pull request, or even before pushing to your fork. This will allow the developers to pre-review your changes and make sure the build will pass so that the changes can be merged as quickly as possible once they are made public.
+
+# Pull requests
+- All PRs should target `dev`, not `master`.
+- It is strongly recommended that you [run `flake8`, `pylint`, `isort`](docs/developing/styleguide.rst#what-is-enforced-in-the-build), and [the test suite](docs/developing/testing.rst#running-tests) to ensure that the build will pass.
+- If your PR closes an issue, include "Closes #XXX" or similar in the messages of the commits that close each issue so the issues will be [automatically closed](https://help.github.com/en/articles/closing-issues-using-keywords) when the commits are merged into `master`.
+  Note that including this text in your PR's description will have no effect because the PR will be merged into `dev`, not `master`, so GitHub does not close the issue. You must add the auto-closing keywords to the *commit* messages.
+- Keep changes minimal.
+- Try not to introduce bugs.
+- Before you commit, make a final pass and make sure you didn't add something in for debugging and forget to take it out.
```

Comment:
It would be nice to add some additional notes:
```
- An Ion maintainer will try to review your PR as soon as possible, but please realize that they are human and have other responsibilities.
- Do not be discouraged if the maintainers  require multiple rounds of review -- we want to make Ion the best it can be.
```